packages/mongo-nav/src/mongo-select/OrgSelect.tsx

```diff
@@ -377,11 +379,17 @@ function OrgSelect({
           >
             {disabled ? 'All Organizations' : current?.orgName ?? ''}
           </span>
-          <Icon
-            size="small"
-            glyph={open ? 'CaretUp' : 'CaretDown'}
-            className={cx(caretBaseStyle, { [iconLoadingStyle]: loading })}
-          />
+          {open ? (
+            <CaretUpIcon
+              size="small"
+              className={cx(caretBaseStyle, { [iconLoadingStyle]: loading })}
```

Comment:
Can this className value be moved to a variable? We're repeating it twice.

Reply:
👍